### PR TITLE
Blog post on Beam graduation

### PR DESCRIPTION
**Pull request by davorbonaci**
R: @francesperry

### CODE REVIEW

File: src/_posts/2017-01-10-beam-graduates.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+layout: post
+title:  "Apache Beam established as a new top-level project"
+date:   2017-01-10 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - davor
+---
+
+Today, the Apache Software Foundation [announced](https://blogs.apache.org/foundation/entry/the-apache-software-foundation-announces)
```
Comment (francesperry) [inline src/_posts/2017-01-10-beam-graduates.md]: Publication blocker: confirm link is correct and live.

File: src/_posts/2017-01-10-beam-graduates.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+that Apache Beam has successfully graduated from incubation, becoming a new Top
+Level Project at the foundation and signifying that its "community and products
```
Comment (francesperry) [inline src/_posts/2017-01-10-beam-graduates.md]: Top-Level?

File: src/_posts/2017-01-10-beam-graduates.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Going forward, we will continue to extend the core abstractions to distill
+additional complex data processing patterns into intuitive APIs, and, at the
+same time, enhance the ability to interconnect additional storage/messaging
+systems and execution engines. Together, we are excited to push forward the
+state of the art in distributed data processing.
+
+This is also an opportunity to thank those who helped and supported us reaching
+this milestone. None of this would be possible without the Apache Software
+Foundation and it’s amazing volunteers who have wholeheartedly welcomed us into
```
Comment (francesperry) [inline src/_posts/2017-01-10-beam-graduates.md]: it's -> its

File: src/_posts/2017-01-10-beam-graduates.md (hunk 2 above)
Comment (davorbonaci) [inline src/_posts/2017-01-10-beam-graduates.md]: Done.

File: src/_posts/2017-01-10-beam-graduates.md (hunk 3 above)
Comment (davorbonaci) [inline src/_posts/2017-01-10-beam-graduates.md]: Done.

File: src/_posts/2017-01-10-beam-graduates.md (hunk 1 above)
Comment (davorbonaci) [inline src/_posts/2017-01-10-beam-graduates.md]: Done.

**Comment by davorbonaci** [comment]
retest this please

**Pull request closed without merging**